src/YieldDistributor.sol

```diff
@@ -189,13 +193,19 @@ contract YieldDistributor is IYieldDistributor, OwnableUpgradeable {
         BREAD.claimYield(BREAD.yieldAccrued(), address(this));
         lastClaimedBlockNumber = block.number;
         uint256 balance = BREAD.balanceOf(address(this));
+        uint256 _minProjectBalance = minProjectBalance;
         uint256 _fixedYield = balance / yieldFixedSplitDivisor;
         uint256 _baseSplit = _fixedYield / projects.length;
         uint256 _votedYield = balance - _fixedYield;
 
         for (uint256 i; i < projects.length; ++i) {
+            address _project = projects[i];
+            uint256 _projectBalance = BREAD.balanceOf(_project);
+            if (_projectBalance < _minProjectBalance) {
+                continue;
```

Comment:
```suggestion
                BREAD.transfer(_project, _baseSplit);
                continue;
```

adding this suggestion as an additional requirement we have recently resolved to 👍

Reply:
Done! [7852dbf](https://github.com/BreadchainCoop/breadchain/pull/95/commits/7852dbf0c30d8d48038ae67828b35de516783e24)